#!/usr/bin/python3
"""Raising a NameError exception with a message"""
def raise_exception_msg(message=""):
    raise NameError(message)
